routes/bitcoin.go

```diff
@@ -0,0 +1,219 @@
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"../db/models"
+	"../helpers"
+	"github.com/gorilla/mux"
+)
+
+// BitcoinData -> this structure is the base response for the project for bitcoins
+type BitcoinData struct {
+	Success bool             `json:"success"`
+	Data    []models.Bitcoin `json:"data"`
+	Errors  []string         `json:"errors"`
+}
+
+// ResponseData -> this structure is used to adapt to the response sent by the coinbase api
+type ResponseData struct {
+	Data models.Bitcoin `json:"data"`
+}
+
+// CreateBitcoin -> Manually does a entry in the db
+func CreateBitcoin(w http.ResponseWriter, req *http.Request) {
+	bodyBitcoin, success := helpers.DecodeBody(req)
+	if success != true {
+		http.Error(w, "could not decode body", http.StatusBadRequest)
+		return
+	}
+
+	var data = BitcoinData{Errors: make([]string, 0)}
+	if !helpers.IsValidBase(bodyBitcoin.Base) {
+		data.Success = false
+		data.Errors = append(data.Errors, "invalid base")
+
+		json, _ := json.Marshal(data)
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		w.Write(json)
+		return
+	}
+
+	btc, success := models.InsertBitcoinValue(bodyBitcoin.Base, bodyBitcoin.Currency, bodyBitcoin.Amount)
+	if success != true {
+		data.Errors = append(data.Errors, "could not create Entry")
```

Comment:
It seems that we would end up sending a 200 OK with an error, we should fix that.